pkg/kubelet/kubelet.go

```diff
@@ -2953,6 +2954,51 @@ func disallowResizeForSwappableContainers(runtime kubecontainer.Runtime, desired
 	return false, ""
 }
 
+// PodResourceSummary save container and pod-level resources for resizing
+type PodResourceSummary struct {
+	// TODO: Add pod-level resources here once resizing pod-level resources is supported
+	InitContainers map[string]v1.ResourceRequirements `json:"InitContainers,omitempty"`
+	Containers     map[string]v1.ResourceRequirements `json:"Containers,omitempty"`
```

Comment:
json tags should start with lower case for consistency with the API.
```suggestion
	InitContainers map[string]v1.ResourceRequirements `json:"initContainers,omitempty"`
	Containers     map[string]v1.ResourceRequirements `json:"containers,omitempty"`
```